glotaran/project/scheme.py

```diff
@@ -50,6 +50,10 @@ class Scheme:
     saving: SavingOptions = SavingOptions()
     result_path: str | None = None
 
+    def __post_init__(self):
+        if len(self.data) == 1 and self.group is None and int(self.group_tolerance) == 0:
```

Comment:
put this out, there are other functions which decide what can be grouped or not.  The user setting for group is always respected (instead for incompatible data, then there will be warning and ungrouped problem will be used). see `model.is_groupable` and `scheme.is_grouped`.